Python/specialize.c

```diff
@@ -1212,6 +1263,34 @@ function_kind(PyCodeObject *code) {
     return SIMPLE_FUNCTION;
 }
 
+static uint32_t
+maybe_function_get_version(PyObject *o, int expected_argcount, int opcode)
```

Comment:
Why the `maybe_` prefix? I would find this clearer as `get_function_version`.
Either way, please add a comment stating that returning zero is a failure.

Reply:
`maybe_` indicates this can accept a non-`PyFunctionObject` (in which case it fails and returns 0. But I can change the name since it doesn't seem to be helping clarity.